apps/prairielearn/src/pages/administratorSettings/administratorSettings.html.ts

```diff
@@ -88,6 +90,18 @@ export function AdministratorSettings({ resLocals }) {
               </script>
             </div>
           </div>
+          ${isEnterprise()
```

Comment:
This should probably only be enabled if OpenAI credentials are configured.

Reply:
Changed in ca500a59ac80e2c9083c1bcdf4ef66647a0892e4